.github/renovate.json

```diff
@@ -4,5 +4,15 @@
     "automergeType": "pr",
     "automerge": true,
     "platformAutomerge": true
+  },
+  "dockerfile": {
+    "packageRules": [
+      {
+        "packageNames": [
```

Comment:
Wait, shouldn't it be `matchPackageNames`?

Reply:
You're correct, `packageNames` is the old name, we should avoid using that, fixed.